src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2835,27 +2855,37 @@ static bool createEntity
    * invoke ensureLocationIndex() in anycase, given that it is harmless in the case the collection and index already
    * exits (see docs.mongodb.org/manual/reference/method/db.collection.ensureIndex/) */
   ensureLocationIndex(tenant);
+  ensureDateExpirationIndex(tenant);
 
   if (!legalIdUsage(attrsV))
   {
     *errDetail =
       "Attributes with same name with ID and not ID at the same time "
       "in the same entity are forbidden: entity: [" + eP->toString() + "]";
 
-    oe->fill(SccInvalidModification, *errDetail, "Unprocessable");
+    oeP->fill(SccInvalidModification, *errDetail, "Unprocessable");
     return false;
   }
 
   /* Search for a potential location attribute */
   std::string     locAttr;
   BSONObjBuilder  geoJson;
 
-  if (!processLocationAtEntityCreation(attrsV, &locAttr, &geoJson, errDetail, apiVersion, oe))
+  if (!processLocationAtEntityCreation(attrsV, &locAttr, &geoJson, errDetail, apiVersion, oeP))
   {
     // oe->fill() already managed by processLocationAtEntityCreation()
     return false;
   }
 
+  /* Search for a potential date expiration attribute */
+  mongo::Date_t dateExpiration;
```

Comment:
I understand it would be safer to use:

```
mongo::Date_t dateExpiration = NO_EXPIRATION_DATE
```

(related with a previous comment about adding a NO_EXPIRATION_DATE constant)

Reply:
Maybe the default constructor is using the same as NO_EXPIRATION_DATE by default... but I think it is better to set it explictely in our code.

Reply:
(These comment would apply not only to this place, but also in other places where `mongo::Date_t` objects are initalized)

Reply:
Fixed in faea551